jobbergate-api/jobbergate_api/apps/job_script_templates/models.py

```diff
@@ -37,74 +28,75 @@ class JobScriptTemplate(Base):
         default=text("'{}'::jsonb"),
         server_default=text("'{}'::jsonb"),
     )
-    name: Mapped[str] = mapped_column(String, nullable=False, index=True)
-    description: Mapped[Optional[str]] = mapped_column(String, default="")
-    owner_email: Mapped[str] = mapped_column(String, nullable=False, index=True)
-    created_at: Mapped[datetime] = mapped_column(DateTime, nullable=False, default=functions.now())
-    updated_at: Mapped[datetime] = mapped_column(
-        DateTime,
-        nullable=False,
-        default=functions.now(),
-        onupdate=functions.current_timestamp(),
-    )
 
-    template_files: Mapped[dict[str, "JobScriptTemplateFile"]] = relationship(
+    template_files: Mapped[list["JobScriptTemplateFile"]] = relationship(
```

Comment:
For simplicity and consistency. The WorkflowFile model can't do this because the files all have the same filename. I can change it back if its needed in the CLI.